packages/blitz-auth/turbo.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "extends": ["//"],
+  "pipeline": {
+    "dev": {
+      "dependsOn": ["blitz#build"],
+      "persistent": true,
+      "cache": false
```

Comment:
why is cache disabled for dev?

Reply:
Because what do you want to cache exactly?
Is there a situation where you want `dev` to immediately **FULL TURBO>>>>>>** and exit? 😄

Reply:
I could also probably change `blitz#build` to `^build` if pnpm recognizes it as a local dependency